src/install.js

```diff
@@ -18,11 +18,14 @@ function ensureCallback (plugin) {
   // Otherwise wrap synchronous plugins to match the async flow
   return function (app, options, next) {
     plugin.register(app, options)
+
     next(null)
   }
 }
 
 function installPlugin (next, plugin) {
+  if (!plugin.register) return next
```

Comment:
Yeah. It's for plugins that don't implement register, but want to use
another lifecycle hook.

On Tue, Jan 26, 2016 at 11:38 AM, Lawson Kurtz <notifications@github.com>
wrote:

> In src/install.js
> <https://github.com/vigetlabs/microcosm/pull/94#discussion_r50860451>:
>
> >      next(null)
> >    }
> >  }
> >
> >  function installPlugin (next, plugin) {
> > +  if (!plugin.register) return next
>
> I see now. Just had to read further.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vigetlabs/microcosm/pull/94/files#r50860451>.
>